### ISSUE TITLE
Use source han sans as the font of the CJKTS language system

### ISSUE THREAD

**Issue by louies0623** [helloSystem/hello#435]
Chinese does not look very elegant on the noto word, the following source han sans is more in line with the local writing usage of the CJK T or S language.

![MknZaRCxO7ZTjv5nfAkjCmVJx8F-tRKvM6fk3eOpRrk](https://user-images.githubusercontent.com/44593430/206886034-f594b486-4d52-4c20-883f-adbdfb5e65d0.jpg)

**Comment by louies0623** [comment]
Source Han Serif also need to add

**Comment by probonopd** [comment]
This package? https://www.freshports.org/chinese/sourcehansans-sc-otf/

**Comment by louies0623** [comment]
Yes but it's only in Simplified Chinese, I need is localization, so the following three must also be

TC - https://www.freshports.org/chinese/sourcehansans-tc-otf/
JP - https://www.freshports.org/japanese/sourcehansans-otf/
KR - https://www.freshports.org/korean/sourcehansans-otf/

**Comment by probonopd** [comment]
I think the proper way would be to install these fonts if the user selects a corresponding language.

The way I undertstand it, users of CJK based systems still occasionally use non-CJK fonts and want to have them installed on their systems.

However, non-CJK users cannot do anything with those CJK fonts, which are just considered "font spam" in the font menus.

Hence, we should only install CJK fonts for users who want them.

**Comment by jjuran** [comment]
I'm unable to read any ideogram-based language, but when I do occasionally see CJK text, I'd prefer to see the glyphs as they're meant to be displayed, not missing-character boxes.  (Consider that I might send a screenshot to someone who can read it.)

I'll never use a CJK system font, but I expect to run into ideographic text on the Web now and then.

**Comment by probonopd** [comment]
Someone would need to implement https://github.com/helloSystem/hello/issues/448 before we can start shipping more non-Latin fonts.
